viewer/core/static/js/app/components/containers/chart-container.js

```diff
@@ -128,9 +128,8 @@ class ChartContainer extends React.Component {
     });
 
     this.yScale = d3Scale.scaleLinear()
-                    .domain([0, d3Array.max(this.biggestDatasetToShow, d => d.y)])
-                    .range([this.state.size.innerHeight, 0])
-                    .nice(); // Y axis should extend to nicely readable 0..100
+                    .domain([0, 1]) // 0% to 100%
+                    .range([this.state.size.innerHeight, 0]);
```

Comment:
According to the [`nice()` documentation](https://github.com/d3/d3-scale/blob/master/README.md#continuous_nice):

> Extends the domain so that it starts and ends on nice round values... For example, for a domain of [0.201479…, 0.996679…], a nice domain might be [0.2, 1.0]

It sounds like `nice()` only worked on Distribution Viewer because most of our charts already had a value near 1. In all other cases, we need to manually specify 1 ad the top of the domain.